composable_kernel/include/tensor_operation/blockwise_gemm_xdlops.hpp

```diff
@@ -170,6 +170,25 @@ struct BlockwiseGemmXdlops_k0mk1_k0nk1_m0n0m1n1m2m3m4n2_v1
         return xdlops_gemm.MakeCDescriptor_M0_N0_M1_N1_M2_M3_M4_N2(c_grid_desc_m0_n0_m1_n1_m2_n2);
     }
 
+    template <typename CGridDesc_B_M_N>
```

Comment:
Length dim "b" is always 1. Do you really need a "B" dimension?

Reply:
Currently b is not always 1. If we eliminate this dim in the grid descriptor, we need to add it to the function signature of gridwise gemm.